crates/runtime/src/wasm_store.rs

```diff
@@ -1,18 +1,21 @@
 use wasmer::Store;
 
 /// New fresh `Store`.
+#[cfg(feature = "default-cranelift")]
 #[must_use]
 pub fn new_store() -> Store {
-    // By importing items within `if` blocks rather than at the top of the file,
-    // we don't run the risk of importing stuff that specific features don't
-    // use (which would cause compiler warnings).
-    if cfg!(feature = "default-cranelift") {
-        use wasmer::{Cranelift, Universal};
+    use wasmer::{Cranelift, Universal};
 
-        let compiler_config = Cranelift::default();
-        let engine = Universal::new(compiler_config).engine();
-        Store::new(&engine)
-    } else {
-        panic!("No Wasmer backend available.");
-    }
+    let engine = Universal::new(Cranelift::default()).engine();
+    Store::new(&engine)
+}
+
+/// New fresh `Store`.
+#[cfg(feature = "default-singlepass")]
+#[must_use]
+pub fn new_store() -> Store {
+    use wasmer::{Singlepass, Universal};
+
+    let engine = Universal::new(Singlepass::default()).engine();
+    Store::new(&engine)
```

Comment:
Why the change? The function is defined with both `default-singlepass` and `default-cranelift`; what changes is the implementation body. I think the old code better reflected that.

Reply:
The old code would result in a compilation error:
```
"could not find `Cranelift` in `wasmer`"
```